app/controllers/admin/orders_controller.rb

```diff
@@ -0,0 +1,31 @@
+class Admin::OrdersController < Admin::BaseController
+  def index
+    @orders = Order.all
+    @users = User.all
+    if params[:email].present?
+      @users = [User.find_by(email: params[:email])]
```

Comment:
why an array?

Reply:
@users is a collection and this instance variable is being used in views (@users.each)
If I don't keep this in an array then I would need to apply some extra conditional blocks in the view

Reply:
Will discuss about it